src/app/Upload.svelte

```diff
@@ -16,17 +29,16 @@
         if (item.kind === "file") {
```

Comment:
**[Suggestion]**: for the reason of clean coding, magic string should be avoided as much as possible. I would use enum or create a constant in this case